Dockerfile

```diff
@@ -0,0 +1,105 @@
+
+ENV HOME /home/user
+WORKDIR $HOME
+
+RUN mkdir .config
+RUN chown -R user:user $HOME/.config
+RUN mkdir .vscode
+RUN chown -R user:user $HOME/.vscode
+
+RUN npm install -g postcss@5.0.0
+RUN npm install -g postcss-syntax@0.10.0
+RUN npm install -g eslint
+RUN npm install -g stylelint
+RUN npm install -g prettier
+RUN npm install -g prettier-eslint
+RUN npm install -g prettier-stylelint
+
+RUN su user -p -c "code --install-extension formulahendry.auto-close-tag"
+RUN su user -p -c "code --install-extension NuclleaR.vscode-extension-auto-import"
+RUN su user -p -c "code --install-extension formulahendry.auto-rename-tag"
+RUN su user -p -c "code --install-extension dzannotti.vscode-babel-coloring"
+RUN su user -p -c "code --install-extension formulahendry.code-runner"
+RUN su user -p -c "code --install-extension naumovs.color-highlight"
+RUN su user -p -c "code --install-extension msjsdiag.debugger-for-chrome"
+RUN su user -p -c "code --install-extension EditorConfig.editorconfig"
+RUN su user -p -c "code --install-extension dsznajder.es7-react-js-snippets"
+RUN su user -p -c "code --install-extension dbaeumer.vscode-eslint"
+RUN su user -p -c "code --install-extension brpaz.file-templates"
+RUN su user -p -c "code --install-extension kumar-harsh.graphql-for-vscode"
+RUN su user -p -c "code --install-extension spywhere.guides"
+RUN su user -p -c "code --install-extension ecmel.vscode-html-css"
+RUN su user -p -c "code --install-extension wix.vscode-import-cost"
+RUN su user -p -c "code --install-extension oderwat.indent-rainbow"
+RUN su user -p -c "code --install-extension Zignd.html-css-class-completion"
+RUN su user -p -c "code --install-extension DavidAnson.vscode-markdownlint"
+RUN su user -p -c "code --install-extension PKief.material-icon-theme"
+RUN su user -p -c "code --install-extension gegeke.node-modules-navigation"
+RUN su user -p -c "code --install-extension eg2.vscode-npm-script"
+RUN su user -p -c "code --install-extension christian-kohler.npm-intellisense"
+RUN su user -p -c "code --install-extension esbenp.prettier-vscode"
+RUN su user -p -c "code --install-extension WallabyJs.quokka-vscode"
+RUN su user -p -c "code --install-extension 2gua.rainbow-brackets"
+RUN su user -p -c "code --install-extension vsmobile.vscode-react-native"
+RUN su user -p -c "code --install-extension shinnn.stylelint"
+RUN su user -p -c "code --install-extension cssho.vscode-svgviewer"
+RUN su user -p -c "code --install-extension wayou.vscode-todo-highlight"
+RUN su user -p -c "code --install-extension jpoissonnier.vscode-styled-components"
+RUN su user -p -c "code --install-extension lXSPandora.vscode-styled-components-snippets"
+RUN su user -p -c "code --install-extension formulahendry.auto-complete-tag"
+RUN su user -p -c "code --install-extension hdg.live-html-previewer"
+RUN su user -p -c "code --install-extension clinyong.vscode-css-modules"
+RUN su user -p -c "code --install-extension bierner.markdown-preview-github-styles"
```

Comment:
The same as bierner.markdown-preview-github-styles

Reply:
Well, Thinking about it better, i think the extensions are correct to allow users to creato documentation for theirs projects.